alchemist/alchemist-incarnation-biochemistry/build.gradle

```diff
@@ -6,6 +6,7 @@ sourceSets {
 
 dependencies {
     compile (
+        project(':alchemist-loading'),
```

Comment:
I'm using that to import simulations in tests. In TestBioRect2DEnvironmentNoOverlap.java and in TestEnvironmentNodes (both in src/test/java) I used some simulations (you can read all of them in src/test/resources) to see if everything was working.